x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/risk_scoring_task/task_execution.ts

```diff
@@ -103,7 +103,7 @@ export default ({ getService }: FtrProviderContext): void => {
             );
           });
 
-          it('@skipInServerlessMKI @skipInServerless starts the latest transform', async () => {
+          it.only('@skipInServerlessMKI starts the latest transform', async () => {
```

Comment:
This test doesn't fail in serverless for me, lets see what CI does, I think it was skipped in error.